napi-inl.h

```diff
@@ -298,7 +298,9 @@ inline bool Value::IsNumber() const {
   return Type() == napi_number;
 }
 
-#ifdef NAPI_EXPERIMENTAL
+// currently experimental guard with version of NAPI_VERSION that it is 
+// released in once it is no longer experimental
+#if (NAPI_VERSION > 2147483646) 
```

Comment:
isn't `NAPI_EXPERIMENAL` already defined as `NAPI_VERSION > 2147483646`

Reply:
I am trying to understand this correctly. I also had a similar question in mind when I saw this patch. Why can't we conditionally define `NAPI_EXPERIMENTAL`?